render_files.py

```diff
@@ -71,11 +72,9 @@ def update_bmh_files(ignition_file, cluster_id, inventory_endpoint, token,
         with open(ignition_file, "r") as file_obj:
             data = json.load(file_obj)
             storage_files = data['storage']['files']
-            # since we don't remove file for now, we don't need to iterate through copy
-            # for file_data in storage_files[:]:
-            for file_data in storage_files:
-                # if file_data['path'] == '/etc/motd':
-                #    storage_files.remove(file_data)
+            for file_data in storage_files[:]:
```

Comment:
Now that you remove a file you should iterate a copy

Reply:
Also why do we need to remove it?

Reply:
1) i am iterating a copy
2) motd is actually an update to existing file, and somehow, ignition 3.1 fails on it. Since it is only motd, i choose to remove it, instead of trying to dig deeper into it